Readme.md

```diff
@@ -16,9 +16,11 @@
 If you get it working on Windows, you should make an instruction file and
 submit a pull request!
 
-1. Clone or unzip this repository into your chosen directory
-2. Open a terminal to said directory and type `npm install` (this will create a *large* `node_modules` directory)
-3. Type `gulp` - the app should open in your default browser
++1. Clone or unzip this repository into your chosen directory
++2. Open a terminal to said directory and type `npm install` (this will create a *large* `node_modules` directory)
++2.5. (Windows only) In the terminal window type 'npm install -g gulp'. This allows the CLI to access gulp
```

Comment:
Actually, this is also a requirement for OS X (I just didn't specify). Could you add gulp (linking to the [npm package](https://www.npmjs.com/package/gulp)) to the requirements bullets instead?